voltdb/releasenotes/notes/bugfixesforvoldbintg-148fe27a87621bc5.yaml

```diff
@@ -0,0 +1,28 @@
+---
+prelude: >
```

Comment:
Can you remove this file? We'll generate a changelog entry on the Changelog file when we release this change

Reply:
sure done.